### DISCUSSION TITLE
./code-server: 36: exec: /root/code-server-3.10.2-linux-arm64/lib/node: not found

### DISCUSSION THREAD

**Discussion by 9yaskhan** [coder/code-server#4247] [General]
I was trying to use #VSCode on Android using #Termux but in last step when I run ' ./code-server ' it show some issue(see in screenshot). I have try many times in different-different methods and also I watched so many videos but I found nothing. Please someone help me to fix this issue.😒
![09-26-08 01 46](https://user-images.githubusercontent.com/91087504/134791715-bbaad02f-f6fd-407d-ad85-ab56aa1e5545.jpg)
![09-23-11 12 26](https://user-images.githubusercontent.com/91087504/134791720-9bb42ed3-4c11-4d6b-83f1-4f313de1e2c0.jpg)

**Comment by jsjoeio** [comment]
Hmm two questions:
- What versions of Node/npm do you have installed? Run `node --version` and `npm --version`
- How are you installing code-server? Did you follow [these docs](https://coder.com/docs/code-server/v3.12.0/termux#install)?

**Reply by Lubandilauben** [reply]
This is the perfectly the same  issue I encountered for along time. And uninstalled termux and installed it again and again the same issue.
Node version installed is : 16

**Reply by jsjoeio** [reply]
@Lubandilauben it could be that you're using Node v16. If you can open a complete bug report with repro steps, I can look into it further!

**Reply by Lubandilauben** [reply]
Thanks jsjoeio.
First i downloaded termux from f-droid.com
After installing it , I opened it and ran the command.
"pkg install nodejs".
After I installed ubuntu with the command.
"pkg install wget openssl-tool proot -y && hash -r && wget https://raw.githubusercontent.com/EXALAB/AnLinux-Resources/master/Scripts/Installer/Ubuntu/ubuntu.sh && bash ubuntu.sh".
Then I ran Ubuntu with the command.
"./start-ubuntu.sh"
Next I  downloaded code server with the command.
"wget https://github.com/cdr/code-server/releases/download/v3.10.2/code-server-3.10.2-linux-arm64.tar.gz"
Extracted tallbar with command.
"tar -xvf ./code-server-3.10.2-linux-arm64.tar.gz"
Finally I ran this command.
"cd code-server-3.10.2-linux-arm64 cd bin"
Then 
code server 
and I get this.
https://user-images.githubusercontent.com/91087504/134791720-9bb42ed3-4c11-4d6b-83f1-4f313de1e2c0.jpg
That's it I hope you help me pliz.

**Reply by Lubandilauben** [reply]
And if I try installing it with yarn by adding the command.
"yarn global add code-server".
I get this error message.
![Screenshot_20210926-013454](https://user-images.githubusercontent.com/76255332/135250327-594a6c09-2f4f-48d3-822f-e5ac134d5e07.png)

**Reply by jsjoeio** [reply]
Sorry, I should have been more clear. By opening a bug report, I meant [here](https://github.com/cdr/code-server/issues/new?assignees=&labels=&template=bug-report.md&title=).

Hmm...you can try this but it's not guaranteed to work. `yarn global add code-server  --ignore-engines`. Let me know what happens!

[Reference](https://github.com/cdr/code-server/issues/4258#issuecomment-929826447)

**Reply by Lubandilauben** [reply]
I get this.
![Screenshot_20210926-014830](https://user-images.githubusercontent.com/76255332/135705770-0ce1935f-cb74-454c-ba97-5c03aea9bc94.png)

**Reply by jsjoeio** [reply]
Ah okay. Thanks for the screenshots. Sorry that didn't work. This is getting hairy. Please open a bug report for further support!

**Reply by monikasharma23** [reply]
Is anyone found the solution?
I am facing same problem.

**Reply by jsjoeio** [reply]
Can you try using the latest version of code-server?

**Reply by SaadSidd29** [reply]
Try directly running from the bin location as 
./code-server

**Reply by jsjoeio** [reply]
Did it work?

**Reply by SaadSidd29** [reply]
Yep... We are trying to create a softlink ... but it's not working... So directly I ran it from location
